### DISCUSSION TITLE
My new setup

### DISCUSSION THREAD

**Discussion by traverseda** [STEMinds/STEMinds-Docs#3] [Show and tell]
![image](https://user-images.githubusercontent.com/2125828/170791873-4b8fa273-2d3f-4037-954f-32003e51fdad.png)

*finally* got around to putting this together.

UI using node-red-dashboard

![image](https://user-images.githubusercontent.com/2125828/170791994-42faf795-aa06-4c08-b752-efb0c551050b.png)

**Comment by Zontex** [comment]
It looks amazing! 😍

Any chance you could share the node red dashboard source code? I believe a lot of people could benefit from it! If you would, feel free to send repository and I'll form from STEMinds account. 

Thanks for sharing !

**Comment by traverseda** [comment]
Sure thing, here it is as a gist! https://gist.github.com/traverseda/bc3542d7de6fa1b2d8a47c5825cf0f0c

The dashboard is not honestly that well organized, and it needs a modified firmware to use the "water for 5 seconds" feature. It's actually pretty simple to set up a dashboard like this in node red, but hopefully people find that useful.

**Comment by traverseda** [comment]
![image](https://user-images.githubusercontent.com/2125828/184510612-db5e3d16-b3c8-41bd-9c18-9f17e7bc6d35.png)

**Reply by Zontex** [reply]
@traverseda would you mind taking closer pictures of Eduponics? would love to write an article about that and your project showcase in our STEMinds blog!

**Comment by traverseda** [comment]
![20220827_093534](https://user-images.githubusercontent.com/2125828/187030529-c124eb7f-70c8-49bc-a420-f7a996e81c9d.jpg)
![20220827_093520](https://user-images.githubusercontent.com/2125828/187030530-bd06986b-c3f8-4fde-9891-c9f038460ca7.jpg)
![20220827_093508](https://user-images.githubusercontent.com/2125828/187030531-e35875a2-4425-404d-997d-5041409c5f4e.jpg)
![20220827_093455](https://user-images.githubusercontent.com/2125828/187030532-c61296a7-28dd-408c-86df-e2c010ef4542.jpg)
![20220827_093446](https://user-images.githubusercontent.com/2125828/187030533-91197b10-c15e-40d6-866c-f4ed97170bcc.jpg)
![20220827_093611](https://user-images.githubusercontent.com/2125828/187030557-d18c7d49-6d2c-4bc9-a1a9-2333a1e7b2af.jpg)

@Zontex

**Comment by traverseda** [comment]
Honestly I think calling it a project is a bit much. The plants definitively grow better with consistent watering from the eduponics, and knowing when the soil dries out is great, but mostly you can just stick it into a planter and everything will just work.

Maybe call it a setup?

**Reply by Zontex** [reply]
Setup sounds like a good description! Do you feel eduponics helped you grow your plants more efficiently? Anything you'd wish there could be that isn't currently included?

**Reply by traverseda** [reply]
It's really nice to be able to get hard data on things like how long it takes your soil to dry out. I found that my soil was drying out every 1-3 days! Way more often than I would have thought. It's especially useful for plants that are a bit out of the way, ones that are more difficult to water and check. I have those plants on a pretty high shelf, so getting at them can be a bit of a pain. Now I just refill the water every few weeks instead of lifting a watering can over my head.

I can't say there's anything I wish it could do that it can't, but I do find myself looking for something for outdoor use. I"d love it if someone made a soil moisture sensor that was zigbee-enabled, solar powered, and suitable for outdoor use. I think something like that could be really useful for everything from planter beds to small farms.

But for what I bought the eduponics for it's doing a great job.

**Reply by Zontex** [reply]
@traverseda Your build is featured on Crowd Supply Eduponics page :)  https://www.crowdsupply.com/steminds/eduponics-mini/updates/3d-cad-models-new-website-and-a-project-showcase-automated-irrigation-system

Keep up the great work! I'm thrilled and looking forward to the new firmware you are working on !

**Reply by traverseda** [reply]
Awesome! Looks good! Honestly the main hold up with the firmware is that I don't know quite where I put my second eduponics... Once I find that I expect work to speed up a whole lot. Don't want to experiment on the one I'm using for plants.
